spring-cloud-stream/src/test/java/org/springframework/cloud/stream/function/SourceToFunctionsSupportTests.java

```diff
@@ -88,6 +89,7 @@ public void testComposedFunctionIsAppliedToExistingMessageSource() {
 			OutputDestination target = context.getBean(OutputDestination.class);
 			assertThat(target.receive(1000).getPayload()).isEqualTo(
 				"HELLO FUNCTION:HELLO FUNCTION".getBytes(StandardCharsets.UTF_8));
+			context.close();
```

Comment:
We don't need to do this here because we use `try-with-resource`: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html